scripts/env-data.sh

```diff
@@ -293,10 +293,13 @@ if [ -z "${PROXY_BASE_URL_PARAMETRIZATION}" ]; then
     PROXY_BASE_URL_PARAMETRIZATION=false
 fi
 
-if [ -z "${GEOSERVER_LOG_LEVEL}" ]; then
-    GEOSERVER_LOG_LEVEL=DEFAULT_LOGGING
+if [ -z "${GEOSERVER_LOG_PROFILE}" ]; then
+    GEOSERVER_LOG_PROFILE=DEFAULT_LOGGING
 fi
 
+if [ -z "${GEOSERVER_LOG_DIR}" ]; then
+    GEOSERVER_LOG_DIR=/opt/geoserver/data_dir/logs/geoserver.log
```

Comment:
@alexkuretz isn't this supposed to be
GEOSERVER_LOG_DIR=${GEOSERVER_DATA_DIR}/logs

Reply:
Yes, good catch!